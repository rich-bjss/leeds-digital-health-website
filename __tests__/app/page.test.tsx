import {render, screen} from '@testing-library/react'
import Page from '@/app/page'

jest.mock('../../components/home/hero-post')
jest.mock('../../components/home/headline/headline-content')
jest.mock('../../components/home/upcoming/upcoming-events')
jest.mock('../../components/layout/footer')

describe('Page', () => {
    it('renders the expected home page content', ()=> {
        const {container} = render(<Page />)

        expect(container).toMatchSnapshot()
    })
})