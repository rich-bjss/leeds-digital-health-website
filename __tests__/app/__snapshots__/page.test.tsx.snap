// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders  1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      class="headline-content-mock"
    />
    <div
      class="hero-post-mock"
    />
    <div
      class="upcoming-events-mock"
    />
    <div
      class="footer-mock"
    />
  </div>
</div>
`;

exports[`Page renders a div 1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      class="headline-content-mock"
    />
    <div
      class="hero-post-mock"
    />
    <div
      class="upcoming-events-mock"
    />
    <div
      class="footer-mock"
    />
  </div>
</div>
`;

exports[`Page renders the expected  1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      class="headline-content-mock"
    />
    <div
      class="hero-post-mock"
    />
    <div
      class="upcoming-events-mock"
    />
    <div
      class="footer-mock"
    />
  </div>
</div>
`;

exports[`Page renders the expected home page 1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      class="headline-content-mock"
    />
    <div
      class="hero-post-mock"
    />
    <div
      class="upcoming-events-mock"
    />
    <div
      class="footer-mock"
    />
  </div>
</div>
`;

exports[`Page renders the expected home page content 1`] = `
<div>
  <div
    class="mx-auto"
  >
    <div
      class="headline-content-mock"
    />
    <div
      class="hero-post-mock"
    />
    <div
      class="upcoming-events-mock"
    />
    <div
      class="footer-mock"
    />
  </div>
</div>
`;
